packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts

```diff
@@ -740,7 +740,9 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
       },
       callErrorCallback: async (context, event) => {
         const errorMessage =
-          event.data?.error?.message || event.data?.error?.Message;
+          event.data?.error?.message ||
+          event.data?.error?.Message ||
+          event.data?.message;
```

Comment:
A bit worried abt the deviation in the error related shapes that `event.data` has here, is there potential that we may be missing any and/or there a way that the shape can be standardized?

Reply:
Yeah we're planning to do more standardization of this in this other pr with a breaking change - https://github.com/aws-amplify/amplify-ui/pull/4134

Issue being that with events so far when we have created custom events we have used the pattern `{ data: { error } }` but by default when xstate handles error events they use `{ data: new Error }`